src/AppBundle/Controller/MainController.php

```diff
@@ -17,4 +18,14 @@ public function indexAction()
     {
         return $this->render('app/main/index.html.twig');
     }
+
+    /**
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function  baptismAction()
+    {
+        return $this->render('app/main/baptism.html.twig',array(
+            'baptismSearchType' => $this->createForm(new BaptismSearchType())->createView(),
```

Comment:
can you put the logic above ? Something like : 
 $baptism = new Baptism();
 $form = $this->createForm('AppBundle\Form\BaptismType', $baptism);
and then pass the form to the view with something like : 
  'baptism' => $form->createView()

Plus, I think you misunderstood this page, since you land on it, you already "found" the baptem you want, this is only the landing page where you will buy this particular baptem.